brainscore_language/model_helpers/huggingface.py

```diff
@@ -288,6 +388,8 @@ def _register_hook(self,
         def hook_function(_layer: torch.nn.Module, _input, output: torch.Tensor, key=key):
             # fix for when taking out only the hidden state, this is different from dropout because of residual state
             # see:  https://github.com/huggingface/transformers/blob/c06d55564740ebdaaf866ffbbbabf8843b34df4b/src/transformers/models/gpt2/modeling_gpt2.py#L428
+            if isinstance(output, tuple):
+                output = output[0]
             output = output[0] if len(output) > 1 else output
```

Comment:
I see. You're correct, but I think the edge case is when `output` is a single-element tuple, a case which L393 doesn't handle. This can be handled using the following:
```suggestion
            output = output[0] if isinstance(output, tuple) else output
```
What do you think?

Reply:
👍